### PR TITLE
Insert statements

### PR DESCRIPTION
**Pull request by Sibusiso-M**

### CODE REVIEW

**Comment by DarthNola** [comment]
There's merge conflicts
please sort that out

**Comment by kbm26** [comment]
Besides for merge conflicts, the changes made are good, removing certain alters and inserts since we are only creating tables. 
Once conflicts are solved, merge

**Review by DarthNola — CHANGES_REQUESTED** [review]
This increase of province was sorted by changing  varchar(120)

**Comment by Sibusiso-M** [comment]
closed to resolve conflicts

File: InsertToTable/InsertContactDetailsData.sql
Comment (alfred-malope) [inline InsertToTable/InsertContactDetailsData.sql:1]: there are a few names that contain spaces

'catherine.van der merwe@yahoo.com'
'sebastian.van der merwe@gmail.com'
'zachary.van der merwe@gmail.com'

File: InsertToTable/InsertDocumentData.sql
Comment (alfred-malope) [inline InsertToTable/InsertDocumentData.sql:1]: The columns in the Document table are Transcript and IdentityDocument, It is not [IDFile]

File: InsertToTable/InsertStudentApplicationData.sql
Comment (alfred-malope) [inline InsertToTable/InsertStudentApplicationData.sql:1]: Check the ERD for StudentApplication columns 


Screenshot 2024-02-11 173030

File: InsertToTable/InsertStudentIInfomationData.sql
Comment (alfred-malope) [inline InsertToTable/InsertStudentIInfomationData.sql:1]: Do not insert Age, we will use calculations from birth date

**Comment by Sibusiso-M** [comment]
Address requested changes

File: InsertToTable/InsertContactDetailsData.sql
Comment (Sibusiso-M) [reply InsertToTable/InsertContactDetailsData.sql:1]: Thanks

**Review by DarthNola — APPROVED** [review]

**Pull request merged**